tests/kernel/bigquery_tests.py

```diff
@@ -769,6 +771,109 @@ def test_load_cell(self, mock_get_table, mock_table_load, mock_table_exists,
                                        source_format='csv', csv_options=mock.ANY,
                                        ignore_unknown_values=True)
 
+  @mock.patch('google.datalab.Context.default')
+  @mock.patch('google.datalab.utils.commands.notebook_environment')
+  @mock.patch('google.datalab.bigquery.Table.exists')
+  @mock.patch('google.datalab.bigquery.commands._bigquery._get_table')
+  def test_pipeline_cell(self, mock_get_table, mock_table_exists, mock_environment,
+                         mock_default_context):
+      table = google.datalab.bigquery.Table('project.test.table')
+      mock_get_table.return_value = table
+      mock_table_exists.return_value = True
+      mock_default_context.return_value = google.datalab.Context('test_project_id', None)
+      env = {
+        'foo_query': google.datalab.bigquery.Query(
+          'SELECT * FROM publicdata.samples.wikipedia LIMIT 5')
+      }
+      mock_environment.return_value = env
+
+      args = {'name': 'bq_pipeline_test', 'debug': True}
+      # TODO(rajivpb): The references to foo_query need to be resolved.
+      cell_body = """
+        email: foo@bar.com
+        schedule:
+            start_date: 2009-05-05T22:28:15Z
+            end_date: 2009-05-06T22:28:15Z
+            schedule_interval: '@hourly'
+        input:
+            path: test/path
+            table: project.test.table
+            schema:
+                - name: col1
+                  type: int64
+                  mode: NULLABLE
+                  description: description1
+                - name: col2
+                  type: STRING
+                  mode: required
+                  description: description1
+        transformation:
+            query: foo_query
+        output:
+            billing: 'foo'
```

Comment:
I don't think this should be here. We should think about where environment config goes for the airflow cluster. Seems like it should be tied to the airflow environment.